python/pipeline/pupil.py

```diff
@@ -540,4 +575,998 @@ def make(self, key):
 
         self.insert1(key)
         for ckey in tqdm(contours):
-            self.Ellipse().insert1(ckey, ignore_extra_fields=True)
\ No newline at end of file
+            self.Ellipse().insert1(ckey, ignore_extra_fields=True)
+
+# If config.yaml ever updated, make sure you store the file name differently so that it becomes unique
+@schema
+class ConfigDeeplabcut(dj.Manual):
+    definition = """
+    # Minimal info needed to load deeplabcut model
+    config_path         : varchar(255)          # path to deeplabcut config
+    ---
+    shuffle             : smallint unsigned     # shuffle number used for the trained dlc model. Needed for dlc.analyze_videos
+    trainingsetindex    : smallint unsigned     # trainingset index used for the trained dlc. model. Needed for dlc.analyze_videos
+    """
+
+@schema
+class TrackingMethod(dj.Manual):
```

Comment:
for list of stuff that won't change often we use Lookup tables. We have all other methods in pipeline/shared.py, you should probably put this one there and follow the convention from there:
``` python
@schema
class TrackingMethod(dj.Lookup):
    definition = """
    tracking_method : tinyint                   # method used for tracking
    ---
    name                : varchar(16)               # short name to identify the tracking method
    details             : varchar(255)              # more details
    language            : enum('matlab', 'python')  # implementation language
    """

    contents = [
        [1, 'manual', 'sjsksksl', 'python'],
        [2, 'deeplabcut', 'ssllslsl', 'python'],
    ]
```

Reply:
I understand, but this table serves the same purpose as `SegmentationTask` under meso and reso modules. For each key, it simply tells which method to be used either `manual` = 0 and `deeplabcut` = 1. I think it should stay here with the current table definition